src/lib/duration/iso-string.js

```diff
@@ -50,3 +54,8 @@ export function toISOString() {
         (m ? m + 'M' : '') +
         (s ? s + 'S' : '');
 }
+
+export function toJSON() {
+    // this may not be fully implemented
```

Comment:
For some reason, `.toJSON` was previously mapped to `.toISOString`. I've created a separate function, at least, to handle the difference when the duration is invalid.